packages/algob/src/types.ts

```diff
@@ -349,6 +351,18 @@ export interface ASADeploymentFlags {
   validRounds?: number
 }
 
+export interface ASCDeploymentFlags {
+  creator: Account
```

Comment:
This field is the same as in `ASADeploymentFlags`. Other fields are similar too. Can you have a parent class for these two?

Reply:
Also make sure Robert agrees with this parameter list. 
All of these params were not defined by the spec

Reply:
Martynas is correct. You merged 2 parameters into one:

Everything below `microAlgo` should go into the `payTxParams`:
https://paper.dropbox.com/doc/Algorand-builder-specs-Vcdp0XNngizChyUWvFXfs#:uid=855054373483415843735838&h2=Smart-Contracts-deployment

Reply:
Done